src/common/components/select-repository-search/index.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component, PropTypes } from 'react';
+import styles from './search.css';
+
+export default class SelectRepositorySearch extends Component {
+
+  render() {
+    let id = this.props.id || 'search-repositories';
+    let label = this.props.label || 'Search';
```

Comment:
On visual designs (there are some in comments #715) repos search input doesn't have a label or placeholder, so I would make the defaults empty.

Also search input on design has an icon in it. Maybe we can find such in assets manager, or something similar in juju/maas?